### ISSUE TITLE
Profile photos not importing

### ISSUE THREAD

**Issue by ryanstraight** [keybase/keybase-issues#2666]
Successfully proved Twitter, Facebook, and Github. Clicking "Refresh my photos" results in a neverending "Scraping your identities" and, even though I've proven those three identities, they're still listed on that screen as "Prove my _____ identity." Cleared cache/cookies, logged out/in, etc. Screenshot attached. Thoughts?

![2016-11-15-15-11-58-285-2016-11-15 15_24_47](https://cloud.githubusercontent.com/assets/23483412/20326905/101c7d8e-ab48-11e6-938e-b9dbc52267e3.png)

**Comment by malgorithms** [comment]
hmm, quick q - did it ever work? or did it stop working after one of the identities was added? (and did you add them all at the same time?)

thanks!

**Comment by ryanstraight** [comment]
And of course, as usual, the Twitter image has scraped when I go back to check. 😆 So it seems to have worked sometime overnight. Now when I click on my photo I can choose from the three. 

But to answer your question, it never worked after adding Twitter and Facebook (which I added at the same time). I added Github later in the day, and still nothing. Oddly, it proved Twitter and then notified me it hadn't been proven a couple times. Strange.

Hope that's helpful! And thanks. :)
